data/core/encyclopedia/history.cfg

```diff
@@ -0,0 +1,41 @@
+#textdomain wesnoth-help
+
+[section]
+    id=history
+    title= _ "History"
+    # All factions are going to be covered except for Dunefolk, which don't have enough history to make one at the moment
+    topics=..history,drakes
+    sort_topics=yes
+[/section]
+
+[topic]
+    id=..history
+    title= _ "History"
+    # This will be a main timeline-style text summarizing the history of all races
+    #text= _ ""
+    generator="contents:history"
+[/topic]
+
+[topic]
+    id=drakes
+    title= _ "History of the Drakes"
+    # This is largely based on section from WoF Story files
+    # AH Details not currently included since current version is not yet canon according to Dalas
+    # I have Ashen's Maw at the end, especially as AH is still a mainline candidate, but I avoided AH storyline
+    text= _ "Warning: Major spoilers ahead for Winds of Fate and minor spoilers for The Rise of Wesnoth
+
+The Drakes have a lengthy yet little-known history stretching back many millennia. They are among the oldest of races, descendants of primordial Fire Dragons. Much of drakish history is preserved by the glider caste, who keep millennia of knowledge. Their histories begin millions of years ago, though the oldest stories are believed to be merely ancient myths, and are doubted even by some drakes. Historians of other races tend to mark anything before the exile as mythical tales.
```

Comment:
In case you didn't already know, millennia is plural of millennium == 1000 years. That's quite different from 'millions of years'. Not sure what your intention here is (maybe carried over from the source text in WoF, I can't remember).